### PR TITLE
ccp ghana

### PR DESCRIPTION
**Pull request by abhidg**
- first draft
- update
- update
- update

### CODE REVIEW

**Review by ekamau — APPROVED** [review]
looks good!

**Review by ekamau — CHANGES_REQUESTED** [review]
For the ccp-ghana parser: 
the mappings for subject.pregancy and subject.pregnancy_outcome are the same.
Confirm if that should be the case. The former should be a boolean field ...

File: schemas/dev/parser.schema.json [hunk 1]
```diff
@@ -698,11 +701,11 @@
           },
           "date": {
             "description": "Date of observation, or end date of observation period",
-            "$ref": "#/definitions/fieldMapping"
```
[the remark this answers is not in the document]
Comment (abhidg) [reply schemas/dev/parser.schema.json:701]: mapping allows combinedTypes

**Comment by abhidg** [comment]
|table       	|valid	|total	|percentage_valid|
|---------------|-------|-------|----------------|
|subject       	|2350	|2777	|84.623695% |
|visit         	|1491	|2777	|53.691033% |
|observation   	|71493	|776981	|9.201383% |

## subject

* 217: data.subject_id must be string
* 209: data must contain ['subject_id', 'country_iso3', 'admission_date', 'age', 'sex_at_birth', 'ethnicity', 'pathogen'] properties
* 1: data.pregnancy_gestational_assessment_weeks must be number

## visit

* 837: data must contain ['visit_id', 'subject_id', 'country_iso3', 'start_date', 'outcome', 'date_outcome'] properties
* 361: data.treatment_experimental_agent_type must be array
* 88: data.visit_id must be string

## observation

* 582489: data must contain ['phase', 'date', 'name'] properties
* 122998: data.date must be date
* 1: data.text must be string

**Pull request merged**